CHANGES.md

```diff
@@ -1,3 +1,12 @@
+## 1.4.3 2018-06-12
+
+This is a follow up release that  includes diagnostic tracing for H2 requests. The feature was 
```

Comment:
spacing "that  includes"

wording: "This follow up release includes".   I'd drop the line after it about not making 1.4.2 (not useful in the changelog)